.azure-pipelines/client.yml

```diff
@@ -34,6 +34,12 @@ jobs:
       scriptPath: 'scripts/devops_tasks/build_packages.py'
       arguments: '-d "$(Build.ArtifactStagingDirectory)" "$(build_targeting_string)"'
 
+  - bash: |
+      mv "$(Build.ArtifactStagingDirectory)/dependencies.html" "$(Build.SourcesDirectory)/dependencies.html"
```

Comment:
We can remove this line and change the output location on L29 instead.

Reply:
I was wondering the same thing. Why is it that we need to move the dependencies.html file before and after this twine command? Does it interfere with the twine check?

Reply:
We might also consider putting the dependencies.html into another named artifact. For reports like this we might not want them in the main artifact zip because we might not want to feed all that to our release pipelines that read all the content from these artifact zips.